ctapipe/resources/base_config.yaml

```diff
@@ -111,3 +112,26 @@ ShowerProcessor:
         - [Positive width, "parameters.hillas.width.value > 0"]
         - [enough pixels, "parameters.morphology.n_pixels > 3"]
         - [not clipped, "parameters.leakage.intensity_width_2 < 0.8"]
+
+# The MuonProcessor analyses DL1 images for muons and writes muon parameters.
+# It is only run if DataWriter.write_muon_parameters=True.
+MuonProcessor:
+  # Choose which events should be considered for muon analysis
+  ImageParameterQuery:
+    # quality critieria should be a list of pairs of [name, filter_expression]
+    # The filter expression should contain the variable `p` which are the
+    # image parameters.
+    quality_criteria:
+      - ["min_pixels", "p.morphology.n_pixels > 100"]
+      - ["min_intensity", "p.hillas.intensity > 500"]
+
+  # Choose which rings should be processed to the actual muon parameters
+  RingQuery:
+    # quality critieria should be a list of pairs of [name, filter_expression]
+    # The filter expression can either contain the variable `parameters` 
```

Comment:
This should be part of the trait help, not only be documented in a config file example